### DISCUSSION TITLE
When ist the next stable version released?

### DISCUSSION THREAD

**Discussion by piotrek1668** [amerkoleci/Vortice.Windows#365] [General]
Hi, in about 2 weeks, we want to use the Vortice-Libraries in our product. I would like to use the newest version (with strong file naming), actually it is the version 2.1.34-beta. I would not like to link beta versions so much. When will the next stable version be released approximately?

**Comment by amerkoleci** [comment]
Hi, please use discussions for those types of question, I can bump stable release whether you want, so ping me when you're ready so I can bump new stable version.

**Reply by piotrek1668** [reply]
OK, next time I will create a discussion for such questions. Thanks for the tip. I will get back to you in about 1-2 weeks about the stable version.

**Reply by amerkoleci** [reply]
Version 2.1.40 stable was released!

**Reply by Beyley** [reply]
> Version 2.1.40 stable was released!

There seems to have been a regression in generation in the release, DXGIFactory3.GetAdapterCount1 is missing now (unless i'm missing something?)

**Reply by amerkoleci** [reply]
> > Version 2.1.40 stable was released!
> 
> There seems to have been a regression in generation in the release, DXGIFactory3.GetAdapterCount1 is missing now (unless i'm missing something?)

Those where helper methods, removed in this commit: https://github.com/amerkoleci/Vortice.Windows/commit/6654e951105bd14a0466bae0b4ce504709179d9e
